proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10773,48 +10773,337 @@ function bufferJoin( test )
+
+  test.case = 'to dst - array from src - unroll';
+  var dst = [ 1, 2, 3 ];
+  var src = _.unrollMake( [ 4, 5, 6 ] );
+  var got = _.bufferMove( dst, src );
+  test.is( got === dst );
 
-  test.case = 'dst - array, src - U8x';
+  test.case = 'to dst - array from src - argumentsArray';
+  var dst = [ 1 ];
+  var src = _.argumentsArrayMake( [ 3 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = [ 3 ];
+  test.identical( got, expected );
+  test.is( got === dst );
+
+  /**/
+
+  test.case = 'to dst - array from src - U8x';
   var dst = [ 0, 2, 3 ];
   var src = new U8x( [ 1, 2, 3 ] );
   var got = _.bufferMove( dst, src );
   var expected = [ 1, 2, 3 ];
   test.identical( got, expected );
+  test.is( got === dst );
 
-  test.case = 'dst - array, src - U16x';
+  test.case = 'to dst - array from src - F32x';
   var dst = [ 0, 2, 3 ];
-  var src = new U16x( [ 1, 2, 3 ] );
+  var src = new F32x( [ 1, 2, 3 ] );
   var got = _.bufferMove( dst, src );
   var expected = [ 1, 2, 3 ];
   test.identical( got, expected );
+  test.is( got === dst );
 
-  test.case = ' identical dst - U16x, src - I16x';
-  var dst = new U16x( [ 1, 2, 3 ] );
-  var src = new I16x( [ 1, 2, 3 ] );
+  test.case = 'to dst - array from src - I64x';
+  var dst = [ 0, 2, 3 ];
+  var src = new I64x( _.bigIntsFrom( [ 1, 2, 3 ] ) ); //1n, 2n, 3n
   var got = _.bufferMove( dst, src );
-  var expected = new U16x( [ 1, 2, 3 ] );
+  var expected = [ 1, 2, 3 ];
   test.identical( got, expected );
+  test.is( got === dst );
+
+  /* unroll */
+
+  test.case = 'to dst - unroll from src - array';
+  var dst = _.unrollMake( [ 1, 2, 3 ] );
+  var src = [ 4, 5, 6 ];
+  var got = _.bufferMove( dst, src );
+  var expected = _.unrollMake( [ 4, 5, 6 ] );
+  test.identical( got, expected );
+  test.is( got === dst );
 
-  test.case = 'dst - I16x, src - U16x';
-  var dst = new I16x( [ 1, 2, 3 ] );
-  var src = new U16x( [ 4, 5, 6 ] );
+  test.case = 'to dst - empty unroll from src - empty unroll';
+  var dst = _.unrollMake( [ ] );
+  var src = _.unrollMake( [ ] );
   var got = _.bufferMove( dst, src );
-  var expected = new I16x( [ 4, 5, 6 ] );
+  var expected = _.unrollMake( [  ] );
   test.identical( got, expected );
+  test.is( got === dst );
 
-  test.case = 'dst - U16x, src - I16x';
-  var dst = new U16x( [ 4, 5, 6 ] );
-  var src = new I16x( [ 1, 2, 3 ] );
+  test.case = 'to dst - empty unroll from src - empty unroll';
+  var dst = _.unrollMake( [ ] );
+  var src = _.unrollMake( [ ] );
   var got = _.bufferMove( dst, src );
-  var expected = new U16x( [ 1, 2, 3 ] );
+  var expected = _.unrollMake( [ ] );
   test.identical( got, expected );
+  test.is( got === dst );
 
+  test.case = 'to dst - unroll from src - argumentsArray';
+  var dst = _.unrollMake( [ 1, 2, 3 ] );
+  var src = _.argumentsArrayMake( [ 4, 5, 6 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = _.unrollMake( [ 4, 5, 6 ] );
+  test.identical( got, expected );
+  test.is( got === dst );
+
+  /**/
```

Comment:
Please, use not delimiters. It divides the test group into 2 parts, and it seems that a new group is starting.

Also, delimiters can be saved. Use the test groups, it is special syntax construction.
```js
test.open( 'dst - Array' );

// test cases 

test.close( 'dst - Array' );
```
Test group opens by method `open` and it closes by method `close`. Methods expect String argument.

The real example https://github.com/Wandalen/wTools/blob/master/proto/dwtools/abase/l0.test/Entity.test.s#L1894